src/Compiler/Driver/ParseAndCheckInputs.fs

```diff
@@ -93,13 +93,13 @@ let PrependPathToSpec x (SynModuleOrNamespaceSig (longId, isRecursive, kind, dec
 
 let PrependPathToInput x inp =
     match inp with
-    | ParsedInput.ImplFile (ParsedImplFileInput (b, c, q, d, hd, impls, e, trivia)) ->
+    | ParsedInput.ImplFile (ParsedImplFileInput (b, c, q, d, hd, impls, e, trivia, i)) ->
```

Comment:
We are running out of the alphabet here, please call this something meaningful :) (identifiers?)

Reply:
Well the thing here is that we just want these values transported to the new instance - we don't really want to know what they are or do anything with them. So what is really needed is a language suggestion for a `with` expression for tuples :)

Reply:
Well `i` just evokes iterators...

Reply:
> Well the thing here is that we just want these values transported to the new instance - we don't really want to know what they are or do anything with them. So what is really needed is a language suggestion for a `with` expression for tuples :)

Is this a common issue?
I thing it is very specific to this codebase -> if it bothers anyone, I can generate modules with functions for this.
( it = overusage of tuples with 5 or more items)

E.g.  Tuple8.Change4thItem etc.

Reply:
Yeah I don't know. `Tuple8.Change4thItem` would help with not needing the extra variables, but still doesn't seem greatly readable to me... It almost seems that when you have this problem you probably should have been using records instead of tuples 🤔